flutter-app/lib/labelscreen.dart

```diff
@@ -501,7 +501,12 @@ class LabelEntry extends StatelessWidget {
                         Chip(
                           avatar: CircleAvatar(
                             backgroundColor: Colors.grey.shade800,
-                            child: Text('$totalImages'),
+                            child: Text(
+                              '$totalImages',
+                              style: TextStyle(
```

Comment:
Lets revert this change for now.. we're working on a UI refresh and it'll be nice to make this standard across the codebase.

Reply:
That's great news, waiting for the new UI 👍 I have reverted this change for now.